starter-code/functions-and-arrays.js

```diff
@@ -24,8 +35,12 @@ var longest = findLongestWord(words);
 console.log(longest);
 
 // Calculating a Sum
-function sumArray (array) {
-
+function sumArray (arr) {
+var total = 0;
+numbers.forEach(function(element){
```

Comment:
WATCH OUT! You are using the variable numbers (outside the function) instead of the parameter of the function (arr, at line 38).